src/types/ButtonProps.ts

```diff
@@ -1,6 +1,8 @@
 interface ButtonProps {
-    disabled: boolean;
     title: string;
+    disabled?: boolean;
+    bgColor?: string;
+    toGo?: string;
```

Comment:
I would say that the `toGo` parameter does not belong to any Button, just the one that includes navigation. For that reason, I would create a new type (perhaps one that extends from the `ButtonProps`) exclusively for the navigation button. 

Check out my suggestion on the `AppLinkButton`